compiler/src/main/kotlin/io/github/apl_cornell/viaduct/syntax/syntax.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Aren't IntelliJ files ignored?

Reply:
They should be. Not sure why this one was generated in the first place; in any case, gone in next commit